src/main.rs

```diff
@@ -4,7 +4,7 @@ use env_logger::Env;
 use flate2::Compression;
 use git_version::git_version;
 use grep_cli::{stdout, unescape};
-use gzp::{deflate::Bgzf, ZBuilder};
+use gzp::{ZBuilder, deflate::Bgzfr};
```

Comment:
```suggestion
use gzp::{ZBuilder, deflate::Bgzf};
```

I think this is just a typo, but should fix CI!